pr2_guide/Spot[] data
